### ISSUE TITLE
Homebrew installation

### ISSUE THREAD

**Issue by bensleveritt** [topshell-language/topshell#3]
Not sure if there's an interest for this, but first thought I had was 'is this on brew'?

If there's a general interest, I'm happy to give it a shot.

**Comment by Ahnfelt** [comment]
It's not currently, but you're very welcome. A package that puts the topshell binary on $PATH?

A central requirement would be create an automated a release process, so that the homebrew package can be released automatically in lockstep with the topshell binaries. How do we go about this?

**Comment by bensleveritt** [comment]
That's right, putting it on $PATH for Brew installations. Personally, I'd prefer to keep it there, and managed by brew (and avoid losing binaries).

Automating the release process would be nice. Keeping the release up to date on homebrew is usually just a case of creating a PR with the updated formula. I've not seen an automated implementation, but I'm sure it'd be straight forward enough as part of the release script.

I'll whip up a manual one now, to automate later.

**Comment by bensleveritt** [comment]
Well, somewhat miraculously, I've managed to get the PR together: https://github.com/Homebrew/homebrew-core/pull/43327

Passed the automated tests, just have to see what the Homebrew maintainers think.

**Comment by Ahnfelt** [comment]
Cool! 

One question though - any reason not to use the topshell-macos binary from https://github.com/topshell-language/topshell/releases? It has no dependencies AFAIR.

**Comment by bensleveritt** [comment]
Using the binaries would put the formula into the Cask section of Homebrew (they don't accept prebuilt binaries in Core), and Casks tend to be GUI applications. Not a big deal, but other, curious minds might want to inspect/remix the installation process before installing, and Homebrew facilitates this easily.

**Comment by Ahnfelt** [comment]
Ah, makes sense. Thank you.

**Comment by bensleveritt** [comment]
@Ahnfelt I've had a bit of feedback that the Homebrew formula shouldn't depend on Node 8. Is there a hard reason for dependency on an old copy of Node?

**Comment by bensleveritt** [comment]
A brief investigation suggests that the target for `pkg` is where the node version matters most, and just omitting `node{x}` from the target will build with whatever version is current (which in Homebrew's case will be the latest Node for the build process).

**Comment by Ahnfelt** [comment]
There's no reason other than 8 being what I've been using so far. My guess is that it would work with the latest version.

**Comment by bensleveritt** [comment]
Just tested with the Node@12 and all good AFAICT. 👍

**Comment by Ahnfelt** [comment]
Master is now on Node 12.
